pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -162,7 +162,14 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) error {
 			return err
 		}
 
-		return currentWorkspace(o.Out, newKubeConfig.Clusters[newKubeConfig.Contexts[kcpCurrentWorkspaceContextKey].Cluster].Server, shortWorkspaceOutput(o.ShortWorkspaceOutput), nil)
+		newServerHost = newKubeConfig.Clusters[newKubeConfig.Contexts[kcpCurrentWorkspaceContextKey].Cluster].Server
+
+		if err = o.getAPIBindings(ctx, newServerHost); err != nil {
+			// display the error, but don't stop the current workspace from being reported.
+			_, err = fmt.Fprintf(o.Out, "error checking APIBindings: %v", err)
```

Comment:
the `err` in `_, err` is an ineffectual assignment, right?

Also, I would output all warnings and errors to o.ErrOut

Reply:
Yeah, it is ineffectual; basically the same as doing no assignment.

I can definitely send errors to o.ErrOut.